smart-contracts/test/RelayPool/prove.ethereum.ts

```diff
@@ -8,7 +8,7 @@ import { ethers } from 'hardhat'
 // These tests are not _actually_ for `RelayBridge` but required for RelayBridge to be able to process claims!
 describe('RelayBridge: prove', () => {
   describe('OPstack', () => {
-    it.skip('should succeed at submitting a proof for an OP1 withdrawal', async () => {
```

Comment:
97 days ago (Dec-06-2024 10:17:03 PM +UTC)! That should pass!

Reply:
yes it works. The RPC is the issue...